spec/helpers/govuk_link_helper_spec.rb

```diff
@@ -80,11 +80,40 @@ def url_for(path)
       end
     end
 
-    context "when new_tab: true" do
-      let(:kwargs) { { new_tab: true } }
+    describe "new_tab" do
+      let(:expected_new_tab_attributes) { { href: "/world", class: "govuk-link", target: "_blank", rel: "noreferrer noopener" } }
 
-      specify "the new tab attributes are present on the link" do
-        expect(subject).to have_tag("a", text: "hello", with: { href: "/world", class: "govuk-link", target: "_blank", rel: "noreferrer noopener" })
+      context "when new_tab: true" do
+        let(:kwargs) { { new_tab: true } }
+
+        specify "the link has the new tab attributes and the default 'new tab' text is appended" do
+          expect(subject).to have_tag("a", text: "hello (opens in new tab)", with: expected_new_tab_attributes)
+        end
+      end
+
+      context "when new_tab: '(opens in a new window)'" do
+        let(:kwargs) { { new_tab: '(opens in a new window)' } }
+
+        specify "the link has the new tab attributes and the provided text is appended" do
+          expect(subject).to have_tag("a", text: "hello (opens in a new window)", with: expected_new_tab_attributes)
+        end
+      end
+
+      context "when new_tab: nil" do
```

Comment:
I think I would expect `nil` to do the same as `false` - ie don't open in a new tab, as nil is usually falsy. 

For opening in a new tab but with no text appended, an empty string `""` feels more intuitive to me?

Reply:
@peteryates any thoughts on this? I know nil vs blank is a bone of contention across many languages… 😂

Reply:
Hmm not sure I agree _entirely_ - I've changed it so:

* `new_tab: true` creates link that opens in a new tab with the default text suffix (set in config)
* `new_tab: "(some text)"` creates link that opens in new tab with ` (some text)` on the end
* `new_tab: nil` creates a link that opens in a tab without any extra text
* `new_tab: ""` creates a link that opens in a tab without any extra text

Reply:
Ok good compromise! 😃